src/structured_data.rs

```diff
@@ -62,8 +65,8 @@ impl StructuredData {
         Ok(structured_data)
     }
 
-    /// This is a static function required for computing a name give the tag-type and identifier to
```

Comment:
I'm not entirely sure what you mean. I expanded the text to make it more precise; is the new version correct?